File: angular-project/src/app/person.ts
export class Person {
    public name: string;
    public id: string;

    constructor(name, id) {
        this.name = name;
        this.id = id;
    }

    public setInfo(name, id) {
        this.name = name;
        this.id = id;
    }

    public toString(): string {
        let ret: string;

        ret = '{"id":"' + this.id + '","name":"' + this.name + '"}';

        return ret;
    }
}

File: play-project/app/models/Person.java
package models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import io.ebean.Finder;
import io.ebean.Model;


@Entity
@Table(name="personal")
public class Person extends Model{


	@Id
	@Column(name="id")
	public int id;

	@Column(name="name")
	public String name;

	public static Finder<Long, Person> find = new Finder<>(Person.class);


	public String toString() {
		return "{\"id\":\"" + this.id + "\",\"name\":\"" + this.name + "\"}";

	}
}



